src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -356,6 +356,36 @@ protected function formatToExpectedJson($key, $value)
     }
 
     /**
+     * Asserts that the response JSON contains the given path.
+     *
+     * @param  string $path
+     * @return $this
+     */
+    protected function seeJsonMatchesPath($path)
+    {
+        $response = json_decode($this->response->getContent(), true);
+
+        // Remove heading $. symbols
+        $search = ltrim($path, '$.');
+
+        // Using random string to protect against null values
+        $notFoundString = str_random(6);
```

Comment:
Str::quickRandom(6) would be better here since we don't care about security in this context.